### PR TITLE
configure heroku deployment

### PR DESCRIPTION
**Pull request by rwajon**
#### What does this PR do?
configure Heroku deployment

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (kagaramag) [inline app.json:1]: @rwajon just want to confirm if `develop` branch is deployed automatically.
What about `staging` and how does it work?

**Review by kagaramag — APPROVED** [review]

**Comment by kagaramag** [comment]
@rwajon remember `.env.sample`, if any

File: app.json (hunk 1 above)
Comment (rwajon) [reply app.json:1]: I've configured the Heroku app to automatically put a raised PR in the staging mode

Comment (kagaramag) [reply app.json:1]: Any branch?

**Pull request merged**